src/components/OrderList.tsx

```
@@ -61,17 +110,88 @@ export const OrderList: React.FC<OrderListProps> = ({
+      switch (sortBy) {
+        case "date":
+          comparison = new Date(a.created_at).getTime() - new Date(b.created_at).getTime();
+          break;
+        case "amount":
+          comparison = a.total_amount - b.total_amount;
+          break;
+        case "customer":
+          comparison = a.customer_name.localeCompare(b.customer_name);
+          break;
+        case "status":
+          const statusOrder = { pending: 1, approved: 2, dispatched: 3, delivered: 4, rejected: 5 };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)